pom.xml

```diff
@@ -115,6 +115,12 @@
         <artifactId>jquery-detached</artifactId>
         <version>1.2.1</version>
       </dependency>
+      <dependency>
+        <!-- TODO: Remove when included in BOM -->
```

Comment:
can you add this to line 83 so that it is obvious when a PR to update the BOM comes in that there is more work to do.  TIA.